### PR TITLE
Django Tutorial Part 8: User authentication and permissions

### PR DESCRIPTION
**Pull request by huu-toan**
ready

### CODE REVIEW

**Comment by huu-toan** [comment]
ready

**Comment by huu-toan** [comment]
ready

**Comment by huu-toan** [comment]
ready

**Comment by huu-toan** [comment]
ready

**Comment by huu-toan** [comment]
Ready

**Comment by huu-toan** [comment]
ready

**Comment by huu-toan** [comment]
ready

File: catalog/views.py [hunk 1]
```diff
@@ -39,11 +39,31 @@ class BookListView(generic.ListView):
...
-    return render(request, 'catalog/book_detail.html', context={'book': book})
+class LoanedBooksByUserListView(generic.ListView):
+    model = BookInstance
+    template_name = 'catalog/bookinstance_list_borrowed_user.html'
+    paginate_by = 10
+
+    def get_queryset(self):
+        if self.request.user.is_authenticated:
+            return BookInstance.objects.filter(borrower=self.request.user).filter(status__exact='o').order_by(
+                'due_back')
```
Comment (huongnt-2545) [inline catalog/views.py]:
```suggestion
            return BookInstance.objects.filter(borrower=self.request.user, status__exact='o')
            .order_by('due_back')
```

**Comment by huu-toan** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
mợt pull

**Comment by huongnt-2545** [comment]
merged

**Pull request merged**